docs/references/general/fitacf.md

```diff
@@ -117,52 +114,27 @@ The arrays for the XCF fit will only be present in the record if an xcf is calcu
-| *x_v* | *m/s* | *[num_pts]* | ***float*** | Velocity from fit of XCF | 
-| *x_v_e* | *m/s* | *[num_pts]* | ***float*** | Velocity error from fit of XCF | 
-| *x_w_l* | *m/s* | *[num_pts]* | ***float*** | Spectral width from lambda fit of XCF | 
-| *x_w_l_e* | *m/s* | *[num_pts]* | ***float*** | Spectral width error from lambda fit of XCF | 
-| *x_w_s* | *m/s* | *[num_pts]* | ***float*** | Spectral width from sigma fit of XCF | 
-| *x_w_s_e* | *m/s* | *[num_pts]* | ***float*** | Spectral width error from sigma fit of XCF | 
+| *x_qflg* | **None** | *[num_pts]* | ***char*** | Quality flag of XCF (FitACF v2 only) | 
+| *x_gflg* | **None** | *[num_pts]* | ***char*** | Ground scatter flag of XCF (FitACF v2 only) | 
+| *x_p_l* | *dB* | *[num_pts]* | ***float*** | Power from lambda fit of XCF (FitACF v2 only) | 
+| *x_p_l_e* | *dB* | *[num_pts]* | ***float*** | Power error from lambda fit of XCF (FitACF v2 only) | 
+| *x_p_s* | *dB* | *[num_pts]* | ***float*** | Power from sigma fit of XCF (FitACF v2 only) | 
+| *x_p_s_e* | *dB* | *[num_pts]* |  ***float*** | Power error from sigma fit of XCF (FitACF v2 only) | 
+| *x_v* | *m/s* | *[num_pts]* | ***float*** | Velocity from fit of XCF (FitACF v2 only) | 
+| *x_v_e* | *m/s* | *[num_pts]* | ***float*** | Velocity error from fit of XCF (FitACF v2 only) | 
+| *x_w_l* | *m/s* | *[num_pts]* | ***float*** | Spectral width from lambda fit of XCF (FitACF v2 only) | 
+| *x_w_l_e* | *m/s* | *[num_pts]* | ***float*** | Spectral width error from lambda fit of XCF (FitACF v2 only) | 
+| *x_w_s* | *m/s* | *[num_pts]* | ***float*** | Spectral width from sigma fit of XCF (FitACF v2 only) | 
+| *x_w_s_e* | *m/s* | *[num_pts]* | ***float*** | Spectral width error from sigma fit of XCF (FitACF v2 only) | 
 | *phi0* | *radians* | *[num_pts]* | ***float*** | Phase determination at lag zero of the ACF | 
 | *phi0_e* | *radians* | *[num_pts]* | ***float*** | Phase determination error at lag zero of the ACF | 
 | *elv* | *degrees* | *[num_pts]* | ***float*** | Elevation angle estimate recommended for scientific use |
 | *elv_fitted* | *degrees* | *[num_pts]* | ***float*** | Alternative elevation angle determined from fitted phase (FitACF v3 only) |
 | *elv_error* | *degrees* | *[num_pts]* | ***float*** | Least-square elevation angle error (FitACF v3 only) |
-| *elv_low* | *degrees* | *[num_pts]* | ***float*** | Lowest estimate of elevation angle (FitACF v1-2 only) |
-| *elv_high* | *degrees* | *[num_pts]* | ***float*** | Highest estimate of elevation angle (FitACF v1-2 only) |
-| *x_sd_l* | **None** | *[num_pts]* | ***float*** | Standard deviation of lambda fit of XCF|
+| *elv_low* | *degrees* | *[num_pts]* | ***float*** | Lowest estimate of elevation angle (FitACF v2 only) |
+| *elv_high* | *degrees* | *[num_pts]* | ***float*** | Highest estimate of elevation angle (FitACF v2 only) |
+| *x_sd_l* | **None** | *[num_pts]* | ***float*** | Standard deviation of lambda fit of XCF |
```

Comment:
@ecbland are all of the `(FitACF v2 only)` notes accurate with regards to `fitacfex`, `fitacfex2`, `lmfit`, etc?  I see that some of those fitting libraries call the `FitSetXrng` function - will that populate the `x_` fields as well?

Reply:
@egthomas Thanks, you're correct. I could just remove these algorithm-specific notes "(FitACF vX only)" from the table entirely. The note in this section about the XCF data is probably sufficient. What do you think?

Reply:
@ecbland I think that sounds fair.  It would be easy enough to add it back in if the community expresses a desire for it.